gplately/download.py

```diff
@@ -430,8 +446,9 @@ def get_topology_geometries(self):
         return files
 
 
-    def get_rasters(self, raster_id_string=None, time=None):
-        """Downloads rasters and images from the web into the "gplately" cache.
+    def get_age_grid(self, time=None, filetype="netCDF"):
```

Comment:
I would like the option to download a list of times, e.g.

```python
gdownload.get_age_grid([0,1,2,3,4], filetype="netCDF")
```

Reply:
Actually, we don't even need the `filetype` keyword for agegrids since the user should get a masked numpy array instead of a filename (see below).